src/shell/focus/target.rs

```diff
@@ -155,6 +158,15 @@ impl PointerFocusTarget {
             _ => None,
         }
     }
+
+    pub fn is_xwm(&self, client: &Client) -> bool {
```

Comment:
Calling this function `is_xwm` is a bit confusing when it just compares the client id of the wl_surface to the `client` argument. At least `client` should be named to specify what client it should be?

Though also, `client` is the client id of the `Xwayland` instance, and not really related to the "xwm" specifically, right? (This is different from `KeyboardFocusTarget::is_xwm` here.)

Reply:
Yeah, this was supposed to work like the `KeyboardFocusTarget`, but then I realized I couldn't get the `XwmId` from the `WlSurface`. Renamed to `is_client`.

> Though also, client is the client id of the Xwayland instance, and not really related to the "xwm" specifically, right?

Technically yes, practically the function in `xwayland.rs` using this matches the client using the `XwmId` as you only have one WM per Xwayland instance. So with a less confusing name, this should be good.